src/CameraControls.ts

```diff
@@ -1,4 +1,4 @@
-import type * as _THREE from 'three';
+import * as THREE from 'three';
```

Comment:
Please use three only for type reference😢

Reply:
Well understood,
I will reorder everything accordingly, just give me some times and sorry for the disturbance, this is actually my first PR.